src/TableData.h

```diff
@@ -92,6 +92,11 @@ extern "C"
    * featured but may be incompatible with older Parquet implementations.
    * Default `V1.0`
    *
+   * COMPRESSION (string) - Selects the compression type for Arrow to use
+   * when writing Parquet files. Relevant libraries must be present in path.
+   * Values supported: `UNCOMPRESSED` (default), `SNAPPY`, `GZIP`, `BROTLI`,
+   * `ZSTD`, `LZ4`, `LZ4_FRAME`, `LZO`, `BZ2`, `LZ4_HADOOP`.
+   *

```

Comment:
Could you also add `COMPRESSION` to `docs/reference.md` for both `pq.writeParquet` and `pq.writeParquetFromTable`?